lib/Github/Api/RateLimit.php

```diff
@@ -0,0 +1,44 @@
+<?php
+
+namespace Github\Api;
+
+/**
+ * Get rate limits
+ *
+ * @link   https://developer.github.com/v3/rate_limit/
+ * @author Jeff Finley <quickliketurtle@gmail.com>
+ */
+class RateLimit extends AbstractApi
+{
+    /**
+     * Get rate limits
+     *
+     * @return array
+     */
+    public function getRateLimits()
+    {
+        return $this->get('rate_limit');
+    }
+
+    /**
+     * Get core rate limit
+     *
+     * @return integer
+     */
+    public function getCoreLimit()
+    {
+        $response = $this->getRateLimits();
+        return $response['resources']['core']['limit'];
```

Comment:
@cursedcoder 

You should use `symfony` preset on your [StyleCI config file](https://github.com/KnpLabs/php-github-api/blob/a89996619661bc490a35ef5133defa87a4030e88/.styleci.yml) or at least, add the `return` fixer.